pkg/service/v1alpha1/utils.go

```diff
@@ -26,9 +26,18 @@ const (
 )
 
 var (
-	ValidFSTypes = []string{FSTypeExt2, FSTypeExt3, FSTypeExt4, FSTypeXfs}
+	ValidFSTypes = []string{FSTypeExt4, FSTypeXfs}
```

Comment:
are we don't want to support ext3 ?

Reply:
Let's support only ext4 and xfs for now, since we have the complete testing suite for just these 2 filesystems.